python/raf/distributed/op.py

```diff
@@ -131,8 +131,8 @@ def reduce_scatter(x, computation="sum", rank_list=None):
 
     Parameters
     ----------
-    x : List[Tensor]
-        A list of tensors of equal shape
+    x : Tensor | List[Tensor]
```

Comment:
Please follow the existing type format in this file.
```suggestion
    x : Tensor or List[Tensor]
```